mandatory/3-function-output.js

```diff
@@ -1,14 +1,21 @@
-// Add comments to explain what this function does. You're meant to use Google!
+/*Add comments to explain what this function does. You're meant to use Google!
+
+ This function it's going to take a random number from 0 to 1(excluding ) and it's going to multiply that
```

Comment:
When commenting the code, explain what the end result (instead of explaining what the code is about). In this case, it's better to say something like, "produces random numbers between 0 to 10 (exclusive)"